CHANGELOG.md

```diff
@@ -8,12 +8,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 ## [unreleased]
 
 ### Added
+-   Project Name can be specified for merge filter #394
 
 ### Changed
 
 ### Removed
 
 ### Fixed
+-   Throw a MergeException if project names do not match in MergeFilter #394 
```

Comment:
move to changed, because this is a change in the ui.

Reply:
changed it